### PR TITLE
Create a mobile template for search results

### PR DESCRIPTION
**Pull request by Anto59290**
- [x] Try to use the DSFR if possible here
- [x] Put the correct data at the correct place
- [x] Adapt cols to match the existing design
- [ ] Try to adapt the mobile view to get something closer to the design

### CODE REVIEW

**Comment by Anto59290** [comment]
image

My current status, style is not perfect but could work as is

File: static/scss/style.scss [hunk 1]
```diff
@@ -1139,3 +1138,33 @@ select[name='language'] {
...
 }
+
+@media(min-width : 768px) {
+    .responsive-tabs .tab-content > .tab-pane {
+        display: block;
+    }
+}
+
+.a4a-search-map-area {
+height: 600px !important;
```
Comment (mlvernay) [inline static/scss/style.scss:1149]: Weird, seems not linted

File: static/scss/style.scss [hunk 2]
```diff
@@ -1139,3 +1138,33 @@ select[name='language'] {
   background-size: auto !important;
   background-position: 50% !important;
 }
+
+@media(min-width : 768px) {
```
Comment (mlvernay) [inline static/scss/style.scss:1142]: maybe reuse a var here

File: templates/search/results.html [hunk 3]
```diff
@@ -16,14 +16,12 @@
     {% include "editorial/header.html" with dark=True %}
 {% endblock header %}
 {% block editorial_content %}
-    <h1 class="mt-3 mb-2">
-        <small class="text-muted" style="font-size:.6em">{% translate "Votre recherche" %}</small>
-        <br>
+    <h1 class="mt-3 mb-2 text-muted">
         {% if what %}
-            <span class="sr-only">{% translate "Activité :" %}</span> {{ what }}
+            {% blocktrans %}Résultats pour «{{what}}» - «{{where}}»{% endblocktrans %}
```
Comment (mlvernay) [inline templates/search/results.html:21]: blocktranslate in last django version

File: templates/search/results.html [hunk 4]
```diff
@@ -73,18 +71,44 @@ <h2 class="h4 m-0 mb-3">
...
+            <ul class="fr-tabs__list"
+                role="tablist"
+                aria-label="Résultats de recherche">
+                <li role="presentation">
+                    <button id="tabpanel-404"
+                            class="fr-tabs__tab fr-tabs__tab--icon-left"
+                            tabindex="0"
+                            role="tab"
+                            aria-selected="true"
+                            aria-controls="tabpanel-404-panel">Liste des résultats</button>
```
Comment (mlvernay) [inline templates/search/results.html:84]: missing translatation

File: templates/search/results.html [hunk 5]
```diff
@@ -73,18 +71,44 @@ <h2 class="h4 m-0 mb-3">
...
+                </li>
+                <li role="presentation">
+                    <button id="tabpanel-405"
+                            class="fr-tabs__tab fr-tabs__tab--icon-left"
+                            tabindex="-1"
+                            role="tab"
+                            aria-selected="false"
+                            aria-controls="tabpanel-405-panel">Carte</button>
```
Comment (mlvernay) [inline templates/search/results.html:92]: same

**Comment by mlvernay** [comment]
Works as expected :+1:

**Pull request merged**